### DISCUSSION TITLE
Is there a way to monitor upstream tags/releases?

### DISCUSSION THREAD

**Discussion by micah** [renovatebot/renovate#25669] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.52

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Is there a way to configure Renovate to monitor a project for releases or tags, and *not* monitor all of that project's dependencies? 

I am wondering if its possible to configure renovate to just notify me when there are new releases of an upstream project, based on a git tag, and not discover and manage the dependencies of the project itself. I would like to do this because I'm hosting some projects that are based off releases in github and because I am not the author of the code, and I do not want to fork it, I don't want Renovate to tell me that X dependency has an update, as this is the upstream's responsibility, but rather I'd love it if Renovate could tell me that there is a new tag available. 

### What you tried so far.

I know that I can use Renovate's Docker digest feature to watch for changes there, if the project publishes a Docker image, but not all projects will publish Docker images.

I'm also aware that there is git-refs, and gitlab-tags datasources for renovate, but I don't understand how I would configure my renovate.json to track my version of the repository with a remote's version

**Comment by rarkins** [comment]
There are `github-tags` and `github-releases` datasources too

**Reply by micah** [reply]
Yeah, I think that using one of these might be the way to go, but I don't quite understand how I would do it.

**Reply by OmegaSquad82** [reply]
same here, newbie and having issues with implementing it for my purpose

**Reply by RahulGautamSingh** [reply]
Did you try using a custom manager? And disabling the rest of the managers?
